corelib/calc/crayfish_mesh_calculator.cpp

```diff
@@ -65,6 +76,11 @@ CrayfishMeshCalculator::Result CrayfishMeshCalculator::expression_valid(const QS
 }
 
 CrayfishMeshCalculator::Result CrayfishMeshCalculator::processCalculation()
+{
```

Comment:
Why not just have the function with optional C++ parameter useMask? 

`CrayfishMeshCalculator::Result CrayfishMeshCalculator::processCalculation(const bool useMask = false)`;